app/assets/stylesheets/govuk_publishing_components/components/_back-link.scss

```diff
@@ -1 +1,45 @@
 @import "govuk/components/back-link/back-link";
+
+.govuk-back-link {
+  &[href] {
+    text-decoration: underline;
+    border-bottom: 1px solid transparent;
+
+    &:focus {
+      text-decoration: none;
+    }
+  }
+
+  &:before {
+    content: "";
+    display: block;
+    position: absolute;
+    top: 0;
+    bottom: 1px;
+    left: 3.31px;
+    width: 7px;
+    height: 7px;
+    -webkit-transform: rotate(225deg);
+    -ms-transform: rotate(2255deg);
+    transform: rotate(225deg);
+    border: solid;
+    border-width: 1px 1px 0 0;
+    clip-path: inherit;
+  }
+}
+
+@include govuk-media-query($until: tablet) {
+
+  .govuk-back-link {
+    position: relative;
```

Comment:
What's this positioning for? Looks like it's already defined in the inspector.

Reply:
This applies the same pseudo-element to increase the touch target after the link as used in the breadcrumb PR

Reply:
What I mean is that position: relative is already set on this element - it doesn't need setting again.

Reply:
See what you mean, have removed duplicate